helm/starboard-exporter/templates/_helpers.tpl

```diff
@@ -25,6 +25,17 @@ app.kubernetes.io/version: {{ .Chart.AppVersion | quote }}
 helm.sh/chart: {{ include "chart" . | quote }}
 {{- end -}}
 
+{{/*
+Monitoring labels
+*/}}
+{{- define "labels.monitoring" -}}
+app: {{ include "name" . | quote }}
+{{ include "labels.selector" . }}
+app.kubernetes.io/managed-by: {{ .Release.Service | quote }}
+app.kubernetes.io/version: {{ .Chart.AppVersion | quote }}
+helm.sh/chart: {{ include "chart" . | quote }}
+{{- end -}}
+
```

Comment:
Suggestion: make `labels.common` just be `labels.monitoring` + the team label?

Reply:
That's why I let you do the thinking 😂